### PR TITLE
[WIP] Config settings

### PR DESCRIPTION
**Pull request by Shine-neko**

### CODE REVIEW

File: src/Tempo/Bundle/AppBundle/Menu/AdminMenuBuilder.php [hunk 1]
```diff
@@ -11,7 +11,28 @@
...
 namespace Tempo\Bundle\AppBundle\Menu;
 
+use Tempo\Bundle\AppBundle\Event\MenuBuilderEvent;
+
 class AdminMenuBuilder extends MenuBuilder
 {
+    public function settingMenu()
+    {
+        $menu = $this->factory->createItem('root');
+        $menu->setChildrenAttribute('class', 'nav nav-pills nav-stacked');
```
Comment (amenophis) [inline src/Tempo/Bundle/AppBundle/Menu/AdminMenuBuilder.php]: css classes here ?

File: src/Tempo/Bundle/AppBundle/Controller/OrganizationController.php [hunk 2]
```diff
@@ -100,7 +100,9 @@ public function updateAction(Request $request, Organization $organization)
      */
     public function createAction(Request $request)
     {
-        if (false === $this->isGranted('ROLE_ADMIN')) {
+        $role = $this->get('sylius.settings.manager')->loadSettings('project')->get('right_create_orga');
+
+        if (false === $this->isGranted($role)) {
             throw new AccessDeniedException();
```
Comment (amenophis) [inline src/Tempo/Bundle/AppBundle/Controller/OrganizationController.php]: return $this->createAccessDeniedException(); ?

File: src/Tempo/Bundle/AppBundle/Menu/AdminMenuBuilder.php [hunk 3]
```diff
@@ -11,7 +11,28 @@
...
+            $this->translate('tempo.menu.admin.setting.general'),
+            array('route' => 'admin_configuration_general')
+        );
+
+        $menu->addChild(
+            $this->translate('tempo.menu.admin.setting.project'),
+            array('route' => 'admin_configuration_project')
+        );
+
+        $this->eventDispatcher->dispatch(MenuBuilderEvent::ADMIN_MAIN, new MenuBuilderEvent($this->factory, $menu));
```
Comment (amenophis) [inline src/Tempo/Bundle/AppBundle/Menu/AdminMenuBuilder.php]: Really usefull ?

Comment (Shine-neko) [inline src/Tempo/Bundle/AppBundle/Menu/AdminMenuBuilder.php]: Yes :)

File: src/Tempo/Bundle/AppBundle/Controller/OrganizationController.php (hunk 2 above)
Comment (Shine-neko) [inline src/Tempo/Bundle/AppBundle/Controller/OrganizationController.php]: this not subject of PR. soon

**Pull request merged**